python/fledge/services/common/microservice_management/routes.py

```diff
@@ -58,6 +58,12 @@ def setup(app, obj, is_core=False):
         # Internal refresh cache
         app.router.add_route('PUT', '/fledge/cache', obj.refresh_cache)
 
+        # Service token verification
+        app.router.add_route('POST', '/fledge/service/verify_token', obj.verify_token)
+
+        # Service token refresh
+        app.router.add_route('POST', '/fledge/service/refresh_token', obj.refresh_token)
```

Comment:
better if we keep as,

PUT   /fledge/service/token/refresh
POST /fledge/service/token/verify

Reply:
POST should be used to create objects, neither of these are creating objects.

Reply:
PUT is better to refresh. GET is better to verify as we don't need that explicit payload anymore but auth request header, Otherwise I see no harm in POSTing a request to get verified. 
also see URL constructs